src/components/Sum.jsx

```diff
@@ -1 +1,23 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
+  <>
+    <p>
+      Sum of&nbsp;
+      {a}
+      &nbsp;and&nbsp;
+      {b}
+      &nbsp;is&nbsp;
+      {a + b}
```

Comment:
try to rewrite in 1 line, using [interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
[one more example of interpolation](https://stackoverflow.com/questions/1408289/how-can-i-do-string-interpolation-in-javascript)